aws/resource_aws_organizations_organizational_unit.go

```diff
@@ -99,19 +101,27 @@ func resourceAwsOrganizationsOrganizationalUnitCreate(d *schema.ResourceData, me
 	}
 
 	if err != nil {
-		return fmt.Errorf("Error creating organizational unit: %s", err)
+		return fmt.Errorf("Error creating Organizational Unit: %s", err)
 	}
 	log.Printf("[DEBUG] Organizational Unit create response: %#v", resp)
 
 	// Store the ID
 	ouId := resp.OrganizationalUnit.Id
 	d.SetId(*ouId)
 
+	if v := d.Get("tags").(map[string]interface{}); len(v) > 0 {
```

Comment:
we could alternatively add tags w/in the CreateInput struct above e.g.
```suggestion
	// Create the organizational unit
	createOpts := &organizations.CreateOrganizationalUnitInput{
		Name:     aws.String(d.Get("name").(string)),
		ParentId: aws.String(d.Get("parent_id").(string)),
		Tags:     keyvaluetags.New(d.Get("tags").(map[string]interface{})).IgnoreAws().OrganizationsTags(),
	}
```